sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -3,6 +3,9 @@
 ### 4.29.0-beta.1 (Unreleased)
 
 #### Features Added
+AsyncCacheNonBlocking - Improve availability
```

Comment:
hmm,  I am not sure whether we should put in the change log. The reason being this is an internal implementation change, no public API or customer cared change.